jwst/undersampling_correction/undersampling_correction.py

```diff
@@ -0,0 +1,76 @@
+
+#  Module for undersampling correction
+#
+import logging
+import numpy as np
+
+from ..datamodels import dqflags
+
+log = logging.getLogger(__name__)
+log.setLevel(logging.DEBUG)
+
+
+def undersampling_correction(input_model, signal_threshold):
+    """
+    Short Summary
+    -------------
```

Comment:
Can you remove the `Short Summary` heading. Just keep the one line description below it.